app/controllers/async_info_controller.rb

```diff
@@ -25,16 +24,6 @@ def base_data
     end
   end
 
-  # TODO: Remove these "shell_version", because they are for service worker functionality we no longer need.
```

Comment:
I think we;re safely past mid-March at this point so I removed this here as well as the corresponding route. cc @benhalpern

Reply:
Yeah, looks good to me.